feat: add the hive doc (#222)
SEKOIA-IO/documentation: docs/integrations/thehive.md

# The Hive

## Overview

> [TheHive](https://thehive-project.org/) is a scalable, open source and free Security Incident Response Platform. This integration through a Symphony module dedicated allows you to duplicate SEKOIA.IO alerts in an instance of The Hive.

Reviewer: ```suggestion
[TheHive](https://thehive-project.org/) is a scalable, open source and free Security Incident Response Platform. This integration through a dedicated Symphony module allows you to duplicate SEKOIA.IO alerts in an instance of The Hive.
```

> [TheHive](https://thehive-project.org/) is a scalable, open source and free Security Incident Response Platform. This integration through a Symphony module dedicated allows you to duplicate SEKOIA.IO alerts in an instance of The Hive.

Reviewer 2: Suggestion enhancement:
```suggestion
[TheHive](https://thehive-project.org/) is a scalable, open source and free Security Incident Response Platform. This integration allows you to forward SEKOIA.IO alerts in an instance of The Hive.
```

## Configure

> First of all, you will have to retrieve your organisation name from your The Hive instance (or [Create an organisation - The Hive documentation](https://docs.thehive-project.org/thehive/user-guides/quick-start/#create-an-organisation)). You will also need a [user apikey](https://docs.thehive-project.org/thehive/user-guides/organisation-managers/users-management/) in this organisation with `manageAlert` permission (for example with the `analyst` [profile](https://docs.thehive-project.org/thehive/user-guides/administrators/profiles/). You have now all informations to configure The Hive module and its `Create Alert` action in SEKOIA.IO.

Reviewer: ```suggestion
First of all, you will have to retrieve your organisation name from your The Hive instance or create an organisation (see [The Hive documentation](https://docs.thehive-project.org/thehive/user-guides/quick-start/#create-an-organisation) to learn how to create an organisation).

Next, you will need a [user apikey](https://docs.thehive-project.org/thehive/user-guides/organisation-managers/users-management/) in this organisation with `manageAlert` permission (for example with the `analyst` profile [profile](https://docs.thehive-project.org/thehive/user-guides/administrators/profiles/). 

You now have all informations to configure The Hive module and its `Create Alert` action in SEKOIA.IO.
```

> The Hive playbook example templates are available in SEKOIA.IO : `Create a new playbook` > `Use a template` > Search for TheHive.

Reviewer: ```suggestion
To start your playbook with an example template, go to : `Create a new playbook` > `Use a template` > Search for TheHive.
```

> Or you can create your own on the same basis. A typical playbook to send alerts to a TheHive instance is this kind of chain :

Reviewer: ```suggestion
You can also create your own on the same basis. A typical playbook to send alerts to a TheHive instance will use this kind of chain :
```

# The Hive

## Overview

[TheHive](https://thehive-project.org/) is a scalable, open source and free Security Incident Response Platform. This integration allows you to forward SEKOIA.IO alerts in an instance of The Hive.

## Configure

First of all, you will have to retrieve your organisation name from your The Hive instance or create an organisation (see [The Hive documentation](https://docs.thehive-project.org/thehive/user-guides/quick-start/#create-an-organisation) to learn how to create an organisation).

Next, you will need a [user apikey](https://docs.thehive-project.org/thehive/user-guides/organisation-managers/users-management/) in this organisation with `manageAlert` permission (for example with the `analyst` profile [profile](https://docs.thehive-project.org/thehive/user-guides/administrators/profiles/). 

You now have all informations to configure The Hive module and its `Create Alert` action in SEKOIA.IO.

> To start your playbook with an example template, go to : `Create a new playbook` > `Use a template` > Search for TheHive.

Reviewer 3: ```suggestion
To start your playbook with an example template, go to : "Create a new playbook" > "Use a template" > Search for TheHive.
```

> You can also create your own on the same basis. A typical playbook to send alerts to a TheHive instance will use this kind of chain :

Reviewer 2: I don't know if it's an omission or a choice, but to be consistent with others occurrences of the name of the product:
```suggestion
You can also create your own on the same basis. A typical playbook to send alerts to a The Hive instance will use this kind of chain :
```
